src/admin-views/help-hub/resources-sidebar-has-license-no-consent.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * The template that displays the resources sidebar.
+ *
+ * @var Tribe__Main $main      The main common object.
+ * @var bool $is_opted_in      Whether the user has opted in to telemetry.
+ * @var bool $is_license_valid Whether the user has any valid licenses.
+ */
+
+$stars_icon_url = tribe_resource_url( 'images/icons/stars.svg', false, null, $main );
+$chat_icon_url  = tribe_resource_url( 'images/icons/chat-bubble.svg', false, null, $main );
+$optin_url      = admin_url( 'edit.php?page=tec-events-settings&tab=general-debugging-tab&post_type=tribe_events#tribe-field-opt-in-status' );
```

Comment:
I wonder if we should be "dynamically" building this.

Reply:
I think these are supposed to target the [modal](https://stellarwp.atlassian.net/browse/TEC-5162) anyways.